hooks/post/30-upgrade.rb

```diff
@@ -31,13 +31,15 @@ def import_subscriptions
 end
 
 def elasticsearch_message
-  return true unless Kafo::Helpers.execute('rpm -q elasticsearch')
-
+  if `rpm -q elasticsearch`.gsub!(/\A"|"\Z/, '') != nil
```

Comment:
So after talking on IRC, I see what you're trying to do the gsub on `"` but that's not really part of the empty string - best bet would be to check rpm exit code.

something like this:

```ruby
if `rpm -q elasticsearch` && $?.success?
  # do installed stuff
else
  # skip it
end
```
```